### PR TITLE
Snackbar made

### PR DESCRIPTION
**Pull request by Asypher00**
please check /components/Snackbar.css for cod
Screenshot (15)
e

### CODE REVIEW

File: components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  right: 40vw;
+     font-family : "comic sans ms";
+    background-color : darkgray;
+    display : flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    width : 25rem;
+    padding-top : 0.625rem;
+    padding-bottom : 0.625rem;
+    border-radius : 0.3125rem;
```
Comment (kjais1720) [inline components/snackbar.css]: use pixels for values less than 10px

File: snackbar.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/components/snackbar.css">
+    <title>Document</title>
```
Comment (Sameony) [inline snackbar.html]: add a relevant title to the document

File: components/snackbar.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    border : 0.125rem solid black;
+  }
+  .snack-footer{ font-family : "comic sans ms";
```
Comment (Sameony) [inline components/snackbar.css]: you've already given the same font family to the parent container, so i believe reapplying the font style here is meaningless.

**Review by Sameony — COMMENTED** [review]
suggested a few minor changes

**Pull request merged**